project/Dependencies.scala

```diff
@@ -54,6 +57,8 @@ object Dependencies {
   val workbenchGoogleTests: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV % "test" classifier "tests" excludeAll(excludeWorkbenchUtil, excludeWorkbenchModel)
   val workbenchMetrics: ModuleID =   "org.broadinstitute.dsde.workbench" %% "workbench-metrics" % workbenchMetricsV excludeAll(excludeWorkbenchUtil)
 
+  val sam:ModuleID = "org.broadinstitute.dsde" %% "sam" % samV
```

Comment:
nit: can this be `org.broadinstitute.dsde.sam-client` ?

Reply:
yep, just have to push to artifactory again